### PR TITLE
Add Indonesian localization

### PR DESCRIPTION
**Pull request by danninov**
This PR will add Indonesian localization and solved #2331.

### CODE REVIEW

**Comment by ansh** [comment]
Great, thanks for the contribution! 

I will wait for a bit and see if there are any other Indonesian users who would like to cross-check this translation. At the same time, will hire someone to check the translations! Thanks!

File: src/locale/locales/id/messages.po [hunk 1]
```diff
@@ -0,0 +1,2478 @@
...
+msgid "Display name"
+msgstr "Nama tampilan"
+
+#: src/view/com/modals/EditProfile.tsx:179
+msgid "Display Name"
+msgstr "Nama Tampilan"
+
+#: src/view/com/modals/ChangeHandle.tsx:485
+msgid "Domain verified!"
+msgstr "Domain diverifikasi!"
```
Comment (Thinkbyte1024) [inline src/locale/locales/id/messages.po:698]: I'm suggesting to change this from `diverifikasi` to `terverifikasi`.

File: src/locale/locales/id/messages.po [hunk 2]
```diff
@@ -0,0 +1,2478 @@
...
+msgid "Learn more about this warning"
+msgstr "Pelajari lebih lanjut tentang peringatan ini"
+
+#: src/view/screens/Moderation.tsx:242
+msgid "Learn more about what is public on Bluesky."
+msgstr "Pelajari lebih lanjut tentang apa yang publik di Bluesky."
+
+#: src/view/com/modals/lang-settings/ContentLanguagesSettings.tsx:82
+msgid "Leave them all unchecked to see any language."
+msgstr "Hilangkan centang untuk melihat semua bahasa."
```
Comment (Thinkbyte1024) [inline src/locale/locales/id/messages.po:1124]: Suggesting a change to `Biarkan semua tidak tercentang untuk melihat bahasa apa pun`

File: src/locale/locales/id/messages.po [hunk 3]
```diff
@@ -0,0 +1,2478 @@
...
+msgstr "Publik, daftar yang dapat dibagikan dan dapat berimbas ke feed."
+
+#: src/view/com/modals/Repost.tsx:52
+#: src/view/com/util/post-ctrls/RepostButton.web.tsx:58
+msgid "Quote post"
+msgstr "Kutip post"
+
+#: src/view/com/modals/Repost.tsx:56
+msgid "Quote Post"
+msgstr "Citar una publicación"
```
Comment (Thinkbyte1024) [inline src/locale/locales/id/messages.po:1612]: Should this be the same as line 1608?

File: src/locale/locales/id/messages.po [hunk 4]
```diff
@@ -0,0 +1,2478 @@
...
+msgstr "Posting ulang atau kutip post"
+
+#: src/view/screens/PostRepostedBy.tsx:27
+msgid "Reposted by"
+msgstr "Diposting ulang oleh"
+
+#: src/view/com/modals/ChangeEmail.tsx:181
+#: src/view/com/modals/ChangeEmail.tsx:183
+msgid "Request Change"
+msgstr "Minta Perubahan"
```
Comment (Thinkbyte1024) [inline src/locale/locales/id/messages.po:1722]: Suggesting a change to `Ajukan Perubahan`

File: src/locale/locales/id/messages.po [hunk 5]
```diff
@@ -0,0 +1,2478 @@
...
+
+#: src/view/com/auth/create/Step2.tsx:53
+msgid "Required for this provider"
+msgstr "Diwajibkan untuk provider ini"
```
Comment (Thinkbyte1024) [inline src/locale/locales/id/messages.po:1726]: Should the word `provider` change to `pelayanan`?

File: src/locale/locales/id/messages.po [hunk 6]
```diff
@@ -0,0 +1,2478 @@
...
+msgid "The support form has been moved. If you need help, please<0/> or visit {HELP_DESK_URL} to get in touch with us."
+msgstr "Formulir bantuan telah dipindahkan. Jika Anda butuh bantuan, silakan<0/> atau kunjungi {HELP_DESK_URL} untuk menghubungi kami."
+
+#: src/view/screens/TermsOfService.tsx:33
+msgid "The Terms of Service have been moved to"
+msgstr "Ketentuan Layanan telah dipindahkan ke"
+
+#: src/view/com/util/ErrorBoundary.tsx:35
+msgid "There was an unexpected issue in the application. Please let us know if this happened to you!"
+msgstr "Sepertinya ada masalah di aplikasi. Beri tahu kami jika Anda mengalaminya!"
```
Comment (Thinkbyte1024) [inline src/locale/locales/id/messages.po:2109]: Suggesting a change to `Sepertinya ada masalah di dalam aplikasi. Silahkan beri tahu kami jika Anda mengalaminya!`

**Review by Thinkbyte1024 — COMMENTED** [review]
Hello world. 👋

I've done reviewing the pull request for the translation. However, there are some suggestions that I want to provide which is also available as a "single comment" in this commit.

My suggestion to the word "Feed" should be change to "Umpan", but there is also a consideration where "Kanal" is a good option. "Kanal" can also be referred to the word "Channel". But, since "Umpan" is (perhaps) a common word in my country, I think this might be the better option. And for the word "diverifikasi" would be considered to be change to "terverifikasi".

If there's something missing in this review, feel free to add more or to correct me if there's anything wrong.

Thank you. :-)

File: src/locale/locales/id/messages.po (hunk 6 above)
Comment (Thinkbyte1024) [reply src/locale/locales/id/messages.po]: Yes, I think thats a good suggestion. :-)

**Review by Thinkbyte1024 — COMMENTED** [review]
I've updated a new review, some of them are leftovers from other languages.
Others are considerable suggestions. :-)

File: src/locale/locales/id/messages.po (hunk 2 above)
Comment (ansh) [reply src/locale/locales/id/messages.po:1124]: Has this change been implemented?

File: src/locale/locales/id/messages.po (hunk 1 above)
Comment (ansh) [reply src/locale/locales/id/messages.po]: Has this change been implemented?

File: src/locale/locales/id/messages.po (hunk 4 above)
Comment (ansh) [reply src/locale/locales/id/messages.po]: Has this change been implemented?

File: src/locale/locales/id/messages.po (hunk 5 above)
Comment (ansh) [reply src/locale/locales/id/messages.po:1726]: Has this change been implemented?

File: src/locale/locales/id/messages.po [hunk 7]
```diff
@@ -0,0 +1,2478 @@
...
+
+#: src/view/com/composer/photos/Gallery.tsx:119
+#: src/view/com/composer/photos/Gallery.tsx:180
+msgid "Add alt text"
+msgstr "Tambahkan teks alt"
+
+#: src/view/com/modals/report/InputIssueDetails.tsx:41
+#: src/view/com/modals/report/Modal.tsx:191
+msgid "Add details"
+msgstr "Tambahkan detail"
```
[the remark this answers is not in the document]
Comment (ansh) [reply src/locale/locales/id/messages.po:107]: Has this change been implemented?

File: src/locale/locales/id/messages.po [hunk 8]
```diff
@@ -0,0 +1,2478 @@
...
+msgid "App passwords"
+msgstr "Kata sandi aplikasi"
+
+#: src/view/screens/AppPasswords.tsx:186
+msgid "App Passwords"
+msgstr "Kata sandi Aplikasi"
+
+#: src/view/com/util/forms/PostDropdownBtn.tsx:236
+msgid "Appeal content warning"
+msgstr "Ajukan banding peringatan konten"
```
[the remark this answers is not in the document]
Comment (ansh) [reply src/locale/locales/id/messages.po:189]: Has this change been implemented?

File: src/locale/locales/id/messages.po [hunk 9]
```diff
@@ -0,0 +1,2478 @@
...
+msgid "Discard"
+msgstr "Buang"
+
+#: src/view/com/composer/Composer.tsx:137
+msgid "Discard draft"
+msgstr "Buang draf"
+
+#: src/view/screens/Moderation.tsx:207
+msgid "Discourage apps from showing my account to logged-out users"
+msgstr "Cegah aplikasi untuk menampilkan akun saya ke pengguna yang tidak login"
```
[the remark this answers is not in the document]
Comment (ansh) [reply src/locale/locales/id/messages.po:682]: Should we commit this change?

**Comment by ansh** [comment]
@danninov @mary-ext @Thinkbyte1024 How do we feel about the state of this localization right now? Are there any pending discussions to be resolved? Or is it good to be merged?

**Comment by ansh** [comment]
@danninov Do you use other apps in Indonesian? For example, I did the translations for Hindi and I kept a lot of english words because the translations weren't used often in other apps. Would be good to check other apps.

**Comment by ansh** [comment]
@danninov And do those apps use the word "Feed"?

**Review by Thinkbyte1024 — COMMENTED** [review]
All good. No more reviews from me. 👍

File: src/locale/locales/id/messages.po [hunk 10]
```diff
@@ -0,0 +1,2478 @@
...
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: @lingui/cli\n"
+"Language: id\n"
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: danninov\n"
+"Language-Team: danninov\n"
```
Comment (ansh) [inline src/locale/locales/id/messages.po:13]: Thanks for all your hard work @danninov @Thinkbyte1024 Please add yourself to the Language Team here

Comment (Thinkbyte1024) [reply src/locale/locales/id/messages.po]: Just a minor suggestion: Changing my username to all lowercase. :-)

`thinkbyte1024`

**Review by ansh — APPROVED** [review]
Looks good!

**Pull request merged**